services/provider/server/server.go

```diff
@@ -224,10 +225,17 @@ func (s *OCSProviderServer) GetStorageConfig(ctx context.Context, req *pb.Storag
 			return nil, err
 		}
 
+		clientMappingConfig, err := s.getClientMapping(ctx)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
why would we fail this function if there is not storage client id mapping config?

Reply:
This function is failing in cases expect isNotFound